pkg/controllers/subnet/subnet_controller.go

```diff
@@ -244,3 +266,30 @@ func (r *SubnetReconciler) Start(mgr ctrl.Manager) error {
 	}
 	return nil
 }
+
+func reconcileSubnet(client client.Client, namespace string, q workqueue.RateLimitingInterface) error {
+	subnetList := &v1alpha1.SubnetList{}
```

Comment:
Can we get subnet list from the specific namespace?

Reply:
OK, updated.